src/main/java/com/socrata/datasync/ui/GISJobTab.java

```diff
@@ -0,0 +1,463 @@
+    private JTextField fileToPublishTextField;
+    private JComboBox publishMethodComboBox;
+    private ButtonGroup publishMethodRadioButtonGroup;
+    private JRadioButton soda2Button;
+    private JRadioButton ftpButton;
+    private JRadioButton httpButton;
+    private JPanel publishViaFTPLabelContainer;
+    private JButton browseForControlFileButton;
+    private JPanel controlFileLabelContainer;
+    private JPanel controlFileSelectorContainer;
+
+
+    private JButton generateEditControlFileButton;
+    private ControlFileModel controlFileModel;
+    private DatasetModel datasetModel;
+    private JTextField runCommandTextField;
+
+    private boolean usingControlFile;
+
+    // build Container with all tab components populated with given job data
+    public GISJobTab(GISJob job, JFrame containingFrame) {
+        mainFrame = containingFrame;
+
+        // build tab panel form
+        jobPanel = new JPanel(new GridLayout(5,2));
+        addFileToPublishFieldToJobPanel();
+        addDatasetIdFieldToJobPanel();
+        addPublishMethodFieldToJobPanel();
+        addRunCommandFieldToJobPanel();
+
+
+        loadJobDataIntoUIFields(job);
+    }
+    
+    private void addRunCommandFieldToJobPanel() {
+        jobPanel.add(UIUtility.generateLabelWithHelpBubble(
+                "Step 4 - Copy command for later (optional)", RUN_COMMAND_TIP_TEXT, HELP_ICON_TOP_PADDING));
+        JPanel runCommandTextFieldContainer = new JPanel(FLOW_RIGHT);
+        runCommandTextField = new JTextField(DEFAULT_RUN_JOB_COMMAND);
+        runCommandTextField.setPreferredSize(new Dimension(
+                JOB_COMMAND_TEXTFIELD_WIDTH, JOB_TEXTFIELD_HEIGHT));
+        runCommandTextField.setEditable(false);
+        runCommandTextField.addMouseListener(new JobCommandTextFieldListener());
+        runCommandTextFieldContainer.add(runCommandTextField);
+        JButton copyJobCommandButton = new JButton(COPY_TO_CLIPBOARD_BUTTON_TEXT);
+        copyJobCommandButton.addActionListener(new CopyJobCommandListener());
+        runCommandTextFieldContainer.add(copyJobCommandButton);
+        jobPanel.add(runCommandTextFieldContainer);
+    }
+    
+    /* GIS Only has a replace method */
+    private void addPublishMethodFieldToJobPanel() {
+        jobPanel.add(UIUtility.generateLabelWithHelpBubble(
+                "Step 3 - Select update method", PUBLISH_METHOD_TIP_TEXT, HELP_ICON_TOP_PADDING));
+        JPanel publishMethodTextFieldContainer = new JPanel(FLOW_RIGHT);
+        String[] onlyOption = {"replace"};
+        JComboBox publishMethodComboBox = new JComboBox(onlyOption);
+        publishMethodComboBox.addActionListener(new PublishMethodComboBoxListener());
+
+        publishMethodTextFieldContainer.add(publishMethodComboBox);
+
+        //Create the radio buttons
+        //TODO: For test purposes only.  Remove these eventually
+        publishViaFTPLabelContainer = new JPanel(FLOW_LEFT);
+        jobPanel.add(publishMethodTextFieldContainer);
```

Comment:
Oh...  this comment was copied wholesale from IntegrationJobTab...

Fascinating.